server/block/slime.go

```diff
@@ -0,0 +1,21 @@
+package block
+
+// A Slime block is a translucent block that entities can bounce on
+type Slime struct {
```

Comment:
This should be called `SlimeBlock` and the file should be renamed to `slime_block.go`